codecov_cli/commands/report.py

```diff
@@ -1,10 +1,7 @@
 import logging
-import uuid
 
 import click
 
-from codecov_cli.fallbacks import CodecovOption, FallbackFieldEnum
```

Comment:
seems that the tests are failing because `CodecovOption` stopped being imported here

Reply:
Thanks @joseph-sentry